components/Common/ProgressionSidebar/ProgressionSidebarIcon/index.tsx

```diff
@@ -0,0 +1,29 @@
+import classNames from 'classnames';
+import type { FC } from 'react';
+
+import styles from './index.module.css';
+
+type ProgressionSidebarIconProps = {
+  isActive?: boolean;
+};
+
+const ProgressionSidebarIcon: FC<ProgressionSidebarIconProps> = ({
```

Comment:
Not sure how I feel about styles and classnames on the SVG, I've much rather have this SVG have 0 logic, and the colour simply be `fill="currentColor"` and then the component using this SVG to actually style it.

Otherwise we are kinda limiting a SVG to be used on only one scenario